pages/specification/migration/_index.md

```diff
@@ -3,5 +3,78 @@ title: Migrating from older drafts
+| extends               | Validation             | 03                 | Yes     | No                            |
+| format                | Annotation             | 02                 | No      | -                             |
+| id                    | Core                   | 04                 | No      | Updated to $id                |
+| if                    | Validation             | 07                 | No      | No                            |
+| items                 | Applicator             | 2019-09            | No      | Updated to prefixItems        |
+| maxContains           | Validation             | 2019-09            | No      | No                            |
+| maxProperties         | Validation             | 04                 | No      | No                            |
+| maximumCanEqual       | Validation             | 02                 | No      | Updated to exclusiveMaximum   |
+| minimumCanEqual       | Validation             | 02                 | No      | Updated to exclusiveMinimum   |
+| minContains           | Validation             | 2019-09            | No      | No                            |
+| minProperties         | Validation             | 04                 | No      | No                            |
+| multipleOf            | Validation             | 04                 | No      | Updated from divisibleBy      |
+| not                   | Validation             | 04                 | No      | No                            |
+| oneOf                 | Validation             | 04                 | No      | No                            |
+| optional              | Validation             | 02                 | No      | Updated to (required)         |
+| pattern               | Core                   | 03                 | No      | No                            |
+| patternProperties     | Validation             | 03                 | No      | No                            |
+| prefixItems           | Applicator             | 2020-12            | No      | Updated from items            |
+| propertyNames         | Validation             | 06                 | No      | No                            |
+| readOnly              | Validation             | 07                 | No      | No                            |
+| required              | Validation             | 03                 | No      | No                            |
+| requires              | Validation             | 02                 | Yes     | -                             |
+| title                 | Annotation             | 01                 | No      | No                            |
+| then                  | Validation             | 07                 | No      | No                            |
+| type                  | Validation             | 02                 | No      | No                            |
+| unevaluatedItems      | Applicator, Annotation | 2019-09            | No      | No                            |
```

Comment:
There is no "Annotation" specification. Only "Core" and "Validation"

Reply:
Thank you, I will effect the changes.